about.html

```diff
@@ -0,0 +1,154 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>JavaScript Programming Language Course</title>
+    <link rel="stylesheet" href="./css/about.css" />
+    <script type="module" src="./js/about.js"></script>
+  </head>
+  <body>
+    <div class="header-bar desktop-visible">
+      <ul>
+        <li>
+          <a href="/"
+            ><img src="./img/socials/facebook.png" alt="Contact us on Facebook"
+          /></a>
+        </li>
+        <li>
+          <a href="/"
+            ><img src="./img/socials/twitter.png" alt="Follow us on Twitter"
+          /></a>
+        </li>
+        <li><a href="/">English</a></li>
+        <li><a href="/">My Page</a></li>
+        <li><a href="/">Logout</a></li>
+      </ul>
+    </div>
+
+    <header>
+      <img
+        id="burger-menu"
+        src="./img/burger/burger_menu.png"
+        alt="Burger menu icon"
+        class="mobile-visible"
+      />
+      <a href="/"
+        ><img
+          src="./img/logo.png"
+          alt="JavaScript Programming Language Course Logo"
+      /></a>
```

Comment:
Kindly make sure that your links are working properly 📝 In the about page, the logo is getting and error rather than predicting the user to home 😥

![image](https://user-images.githubusercontent.com/15807343/182003417-247fed50-0734-4666-9f47-a552f23c381b.png)